portfolio/src/main/webapp/css/base/utility.css

```diff
@@ -0,0 +1,41 @@
+/*
+    This file contains a series of utility styling classes.
+    These can be combined with other classes to make quick, consistent adjustments to style.
```

Comment:
It would be helpful to document each block in this file. Unlike the other files, these declarations don't share a common theme, and it's hard to know how they should be used based on just the name.

Reply:
Agreed - added